tests/test_ui/test_ui_drag_and_drop.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from playwright.sync_api import sync_playwright
+
+@pytest.fixture(scope="module")
```

Comment:
This is duplicated across all files. Put it in its own file.

Reply:
Thank you for the feedback! I'll move the `browser` fixture to a separate file to avoid duplication across the test files. I'll update the PR accordingly.